### ISSUE TITLE
ggsave legend issues

### ISSUE THREAD

**Issue by dhicks** [yhat/ggplot#463]
MWE:  
```
import pandas as pd
from ggplot import *

meat_lng = pd.melt(meat[['date', 'beef', 'pork', 'broilers']], id_vars='date')
thing = ggplot(aes(x='date', y='value', colour='variable'), data=meat_lng) + \
    geom_point() + \
    stat_smooth(color='red')

#print(thing)

ggsave(plot = thing, format = 'png')
ggsave(plot = thing, format = 'pdf')
```

Expected behavior:  The two image files should be identical to [this example](https://camo.githubusercontent.com/b063a65144cd7f48b81437f42a7719270698fb8f/68747470733a2f2f7261772e6769746875622e636f6d2f796861742f6767706c6f742f6d61737465722f6767706c6f742f74657374732f626173656c696e655f696d616765732f746573745f726561646d655f6578616d706c65732f6767706c6f745f6d6561742e706e67), from the `ggplot` for Python readme.  

Actual behavior:  See below.  Both have a ribbon, which I assume has been added to `stat_smooth` since the example was generated.  More importantly, the legend is malformed in the `png` version, with legend items placed over each other.  The `pdf` is legible, but the legend items are a little wider than expected.  

`pdf`:
[283578011.pdf](https://github.com/yhat/ggplot/files/54518/283578011.pdf)

`png`:
![283578011](https://cloud.githubusercontent.com/assets/3300682/11638225/c33b7f6e-9cf3-11e5-96b4-99a98d3d58b0.png)

Python 3.4, Mac OS X El Capitain, ggplot 0.6.8, matplotlib 1.5.0.

**Comment by rbdixon** [comment]
I'm targeting a PDF built at 96 DPI. I had the same problem until I set `dpi=96` and ended up with a correct PNG output. I still think it is a bug but I bet the spacing between the legend entries is expressed in pixels.

**Comment by glamp** [comment]
fixed in 0.9.3
